### ISSUE TITLE
[Bug]

### ISSUE THREAD

**Issue by Le0xFF** [OpenIntelWireless/HeliPort#155]
**Describe the bug**
HeliPort keeps disconnecting and reconnecting to WiFI after a couple of seconds.

**To Reproduce**
1. Open HeliPort
2. Connect to WiFi
3. It will automatically disconnect and reconnect

**Expected behavior**
To keep my connection stable

**Environment (please complete the following information):**
 - OS Version: macOS 10.15.6
 - Software Version: [latest build taken from here](https://github.com/1hbb/OpenIntelWireless-Factory/releases)

**Additional context**
My WiFi card is Intel AC 9260 [8086:2526].
I placed kext in "Others" folder of my Clover EFI partition.
I placed HeliPort in my "/Application" folder.
Everytime the app connects to my wifi, it stays connected for a couple of seconds, then it disconnects and reconnects again and again, making my wifi unusable.
I attached also a log, hoping it will help.

[log.txt](https://github.com/OpenIntelWireless/HeliPort/files/5048269/log.txt)

**Comment by ErrorErrorError** [comment]
Hm, can you please press the option key + HeliPort WiFi Menu Icon and there should be an option that says "Create Diagnostics report..". Upload the zip.

**Comment by Le0xFF** [comment]
Here it is the bug report.

It should include connecting to my router and also connecting to my phone's hotspot. When I try to connect to the latter, it stays connected without any problem, so probably the distance between my PC and my router is a problem.
But it is kinda strange, because I have no such issue with the same wifi card on Windows or Debian on the same PC.

[bugreport_15685.zip](https://github.com/OpenIntelWireless/HeliPort/files/5049759/bugreport_15685.zip)

**Comment by williambj1** [comment]
The are a ton of deauths in your log, are you sure your password is correct?

**Comment by Le0xFF** [comment]
Yes, the password is correct, in fact I can browse the internet for 10 seconds and then it disconnects.
Probably as I said, the distance between my PC and the router could be the problem.

**Comment by linjiyu14** [comment]
These days I also have this problem.
I updated kexts and heliport yesterday, and this problem still exits.
PC: XiaoMi NoteBook Pro 15.6   intel ac8265

bugfile:
[bugreport_35009.zip](https://github.com/OpenIntelWireless/HeliPort/files/5107688/bugreport_35009.zip)

**Comment by mlinton** [comment]
I am trying to create the debug report and I don't get a prompt to save it?  But I'm having the same issues with strange connection and disconnection.
